roles/kafka_broker/tasks/set_principal.yml

```diff
@@ -46,7 +46,7 @@
     kafka_broker_principal: "User:{{ extracted_principal }}"
   vars:
     extracted_principal: "{{distinguished_name_from_keystore.stdout if kafka_broker_final_properties['ssl.principal.mapping.rules'] is not defined \
-                             else distinguished_name_from_keystore.stdout|resolve_principal(kafka_broker_final_properties['ssl.principal.mapping.rules'])}}"
+                             else distinguished_name_from_keystore.stdout|confluent.platform.resolve_principal(kafka_broker_final_properties['ssl.principal.mapping.rules'])}}"
```

Comment:
This is a bug on the released branch, and customers do not have a workaround. I'd just updating 7.1.0-post

Reply:
Yeah, we'll be getting this in post branches shortly. And will probably also pick up some other bug fixes if we are modifying the post branch.